### PR TITLE
Use HeadshotDeath instead of BulletDeath for Umagon's rifle

### PR DESCRIPTION
**Pull request by abcdefg30**
"Follow-up" of #8742, see https://github.com/OpenRA/OpenRA/pull/8742#discussion_r34966625.

### CODE REVIEW

File: mods/ts/weapons/smallguns.yaml [hunk 1]
```diff
@@ -263,5 +263,4 @@ Sniper:
 			Light: 5
 			Heavy: 5
 			Concrete: 5
-		DamageTypes: Prone100Percent, TriggerProne, BulletDeath
-
+		DamageTypes: Prone100Percent, TriggerProne, HeadshotDeath
```
Comment (abcdefg30) [inline mods/ts/weapons/smallguns.yaml]: Somehow I still have another line below this one locally...

Comment (GraionDilach) [inline mods/ts/weapons/smallguns.yaml]: Windows vs Linux line ending issue?

**Comment by GraionDilach** [comment]
:+1: though.

**Comment by penev92** [comment]
For the record, this doesn't look like a bullet-induced death to me (type 3). More like chemically-induced.
But there isn't much point to argue about that here and now so :+1: otherwise.

**Pull request merged**